deploy/nl/embeddings.yaml

```diff
@@ -71,6 +71,11 @@ indexes:
     embeddings: gs://datcom-nl-models/embeddings_undata_ilo_2024_05_15_11_18_05.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
     healthcheck_query: "Employment"
+  undata_dev_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_undata_dev_2024_05_21_15_37_51.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+    healthcheck_query: "Employment"
```

Comment:
Have a TODO to remove post test?

Reply:
Added a todo to `DCNames` definition. Might make sense to keep